src/cbmc_starter_kit/template-for-repository/proofs/Makefile.common

```diff
@@ -341,9 +344,9 @@ UNWINDSET ?=
 # contracts).  To satisfy this requirement, it may be necessary to
 # unwind some loops before the function contract and loop invariant
 # transformations are applied to the goto program.  This variable
-# EARLY_UNWINDSET is identical to UNWINDSET, and we assume that the
-# loops mentioned in EARLY_UNWINDSET and UNWINDSET are disjoint.
-EARLY_UNWINDSET ?=
+# CPROVER_LIBRARY_UNWINDSET is identical to UNWINDSET, and we assume that the
+# loops mentioned in CPROVER_LIBRARY_UNWINDSET and UNWINDSET are disjoint.
+CPROVER_LIBRARY_UNWINDSET ?=
```

Comment:
🍓 This changes a variable that users can set in their Makefiles and thus is likely to break somebody's proof. Why is this change needed?

Reply:
Is this orthogonal to the changes needed for contracts and DFCC? If so, perhaps it could be offered as a separate PR with explanation for the motivation?

Reply:
I believe Mark had a similar concern, so if it could be factored out it might progress more smoothly.

Reply:
Nobody uses this variables in their proofs today. This change is necessary cause we may need to unroll loops for CPROVER library (e.g., models for `strlen` and `memcpy`) at different stages depending whether users will use contracts or not. For instance, using dynamic frames, users need to unroll everything during instrumentation; however, if no contracts are used, then they may unroll these loops when invoking CBMC for property checking.

Reply:
OK